docs/Using Fleet/GitOps.md

```diff
@@ -569,16 +569,44 @@ Can only be configured for all teams (`org_settings`).
 
 #### mdm
 
-The `mdm` section lets you enable MDM features in Fleet.
+##### apple_business_manager
 
-- `apple_bm_default_team` - is name of the team that macOS hosts in Apple Business Manager automatically enroll to when they're first set up. If empty, hosts will enroll to "No team" (default: `""`).
+- `organization_name` - is the organziation name associated with the Apple Business Manager account.
+- `macos_team` - macOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager.
+- `ios_team` - iOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager.
+- `ipados_team` - iPadOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager.
 
 ##### Example
 
 ```yaml
 org_settings:
   mdm:
-    apple_bm_default_team: "Workstations" # Available in Fleet Premium
+    apple_business_manager: # Available in Fleet Premium
```

Comment:
Maintain support for the old `apple_bm_default_team`. 

When the user upgrades, set the new `organization_name.macos_team` to the value for the `apple_bm_default_team`.

If the user sets `apple_bm_default_team` and `organization_name` at the same time, return the following error: TODO

Reply:
Redirect for the URL in the error message is here: https://github.com/fleetdm/fleet/pull/21043/files#diff-0c6120927d3e65309562b5b15b261d3298d05fcb0ff18e5b6512ee683b7ad6b3R533